express_cut/express_service/serializers.py

```diff
@@ -19,3 +20,10 @@ def create(self, validated_data):
         validated_data['password'] = make_password(validated_data.get('password'))
         return User.objects.create(**validated_data)
 
+
+class DailyScheduleSerializer(serializers.ModelSerializer):
```

Comment:
Consider Handling here the timeslots. Lets say I want to PUT a schedule, shouldn't I be able to send in the request the related timeslots?  Or If I wanted to do a GET, shouldn't I receive in the response the timeslots related to that schedule. Something like this:
`
{
    "date": "2020-10-25",
    "stylist_id": 0,
    "time_slot": {
        {
            "start_time": "07:00:00",
            "end_time": "12:00:00"
        },
        {
            "start_time": "13:00:00",
            "end_time": "17:00:00"
        },
    }
}`

Reply:
This is pending. Will work on it in a new issue. (Reference: https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-nested-objects)